packages/snaps-execution-environments/src/common/endowments/hardenedEndowments.ava.test.ts

```diff
@@ -201,21 +203,21 @@ const testSubjects = {
 };
 
 // These are fake types just to make this test work with the TypeScript
+/* eslint-disable @typescript-eslint/naming-convention */
 type HardenedEndowmentSubject = {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
   __flag: unknown;
-  // eslint-disable-next-line @typescript-eslint/naming-convention
   prototype: { __flag: unknown };
 };
+/* eslint-disable @typescript-eslint/naming-convention */
 type HardenedEndowmentInstance = {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
   __flag: unknown;
-  // eslint-disable-next-line @typescript-eslint/naming-convention
   __proto__: { __flag: unknown };
 };
```

Comment:
This will disable the naming convention rule for the rest of the file. You can just put `/* eslint-disable @typescript-eslint/naming-convention */` once before both types, and `/* eslint-enable @typescript-eslint/naming-convention */` once after both types.